src/internal/Expressions.scala

```diff
@@ -50,12 +51,18 @@ trait Expressions extends Utils {
     override final lazy val hashCode = scala.runtime.ScalaRunTime._hashCode(this.asInstanceOf[Product])
   }
 
-  abstract class Stm // statement (links syms and definitions)
+  // statement (links syms and definitions)
+  abstract class Stm {
+    def lhs: List[Sym[Any]] = infix_lhs(this)
+    def rhs: Any = infix_rhs(this)
+    def defines[A](sym: Sym[A]): Option[Def[A]] = infix_defines(this, sym)
```

Comment:
Agreed, it's not clear why infix_ form was used in the first place, since normal inheritance works just fine here.